kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -112,11 +95,16 @@
         </KGrid>
       </div>
     </div>
-    <ConfirmCancellationModal
-      v-if="showConfirmationModal"
+    <KModal
+      v-if="showCloseConfirmation"
```

Comment:
As I began using ConfirmCancellationModal I found it easier to work with to just use KModal directly rather than try to rework it to suit all of the use cases.

Reply:
That seems sensible! As I mentioned above, if it is feeling too verbose, defining handlers inline might help.